Makefile

```diff
@@ -98,23 +97,23 @@ upload:
 
 ## Run test environment
 compose-ES5:
```

Comment:
Minor naming thing, would you mind changing the name of this (and the others) to `compose-ES7`? Maybe it should just simply be called `compose`?

Reply:
My personal opinion is that `compose` makes sense because there is nothing obviously connected to the version here.

Reply:
This has been addressed on #276, which is a duplicate PR that I control.